mne/io/fiff/raw.py

```diff
@@ -61,6 +61,8 @@ class Raw(BaseRaw):
         List of channels' names.
     n_times : int
         Total number of time points in the raw file.
+    times :  ndarray
+        Time vector.
```

Comment:
can you specify it is in seconds and that it starts from 0. independently of first_samp attrubute?